### ISSUE TITLE
Connection to Deemix stopped working

### ISSUE THREAD

**Issue by JoachimVeulemans** [ta264/Lidarr.Plugin.Deemix#17]
When testing the connection in Lidarr, I am getting the following error in the Deemix logs:

deemix3  | 2024-10-03T16:36:00.451Z [error] unhandledRejection: {"UNKNOWN":"Preview is not available"}
deemix3  | GWAPIError: {"UNKNOWN":"Preview is not available"}
deemix3  |     at GW.api_call (/snapshot/deemix-gui/server/dist/app.js)
deemix3  |     at runMicrotasks (<anonymous>)
deemix3  |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
deemix3  | GWAPIError: {"UNKNOWN":"Preview is not available"}
deemix3  |     at GW.api_call (/snapshot/deemix-gui/server/dist/app.js)
deemix3  |     at runMicrotasks (<anonymous>)
deemix3  |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
deemix3  | 

Any idea what it could be?

**Comment by oouhriu** [comment]
Same issue with my setup today. Also been posyed on the Lidarr-On-Steroids repo

**Comment by Stooovie** [comment]
Yes, [this](https://github.com/youegraillot/lidarr-on-steroids/issues/110)

**Comment by JoachimVeulemans** [comment]
Issue seems resolved, both https://gitlab.com/Bockiii/deemix-docker and https://github.com/youegraillot/lidarr-on-steroids seem to be working now.
